src/http/http-native-impl.ts

```diff
@@ -37,11 +47,14 @@ export class HttpNativeImpl extends BaseHttp implements Http {
         body: body as string,
       });
       const text = await response.text();
+      const contentType = response.headers
```

Comment:
This block-scoped variable `contentType` has the same name and role as the variable in line 30.